podcasts/PodcastImageView.swift

```diff
@@ -129,6 +129,7 @@ class PodcastImageView: UIView {
         if let imageView = imageView {
             imageView.backgroundColor = UIColor.clear
             imageView.clipsToBounds = true
+            imageView.contentMode = .scaleAspectFit
```

Comment:
Hey @rviljoen after talking with our product team, we want to keep consistency with the other clients and keep the square format here. Would you mind change this into `scaleAspectFill`?
I can confirm that this change fixes this problem and it works with your previous fix in #3166